kbase-extension/static/kbase/js/widgets/function_output/kbaseReportView.js

```diff
@@ -108,18 +95,21 @@ define([
                 this.options.report_ref
             );
 
-            this.ws
-                .get_objects2({ objects: [this.objIdentity] })
+            return this.ws.get_objects2({ objects: [this.objIdentity] })
                 .then((result) => {
                     this.reportData = result.data[0].data;
-                    return this.getLinks(this.reportData);
+                    return Promise.all([
+                        this.getLinks(this.reportData),
+                        this.getCreatedObjectInfo(this.reportData)
```

Comment:
Move this up here - previously it was another random, unlinked Promise in the render function that could dangle.